client/src/components/Header/Header.scss

```diff
@@ -4,15 +4,153 @@
     url('../../assets/fonts/WTT-THIN.ttf') format('truetype');
 }
 
+// .header {
+//   position: fixed;
+//   top: 0;
+//   width: 100%;
+//   background-color: rgba(20, 125, 22, 0.8);
+//   border-color: transparent;
+
+//   /* put this above the map drawer but below modal dialogs */
+//   z-index: 1250 !important;
+// }
+
 .header {
-  position: fixed;
-  top: 0;
+  font-family: 'Montserrat', sans-serif;
+  display: flex;
+  justify-content: center;
   width: 100%;
-  background-color: rgba(20,125,22, 0.8);
-  border-color: transparent;
+  height: 64px;
+
+  &__navigation {
+    display: flex;
+    width: 70%;
+
+    @media only screen and (max-width: 768px) {
+      width: 90%;
+    }
+
+    &__logocontainer {
+      display: flex;
+      align-items: center;
+      max-width: 272px;
+
+      &__logo {
+        padding-left: 10px;
+        width: 100%;
+      }
+    }
+
+    &__navcontainer {
+      position: relative;
+      display: flex;
+      left: 5%;
+      align-items: center;
+
+      @media only screen and (max-width: 768px) {
+        display: none;
+      }
+
+      &__buttons {
+        margin: 8px 8px;
+        font-size: 1.5rem;
+        border: none;
+        background-color: white;
+        padding: 0;
+        padding-right: 32px;
+
+        a {
+          display: flex;
+          flex-direction: column;
+          color: black;
+
+          &:before {
+            color: black;
+            font-weight: bold;
+            content: attr(Title);
+            visibility: hidden;
+            height: 0;
+          }
+
+          &:hover {
+            color: #3fab45;
+            text-decoration: none;
+            font-weight: bold;
+          }
+        }
+      }
+    }
+
+    &__authcontainer {
+      display: flex;
+      align-items: center;
+      margin-left: auto;
+
+      &__button {
+
+        @media only screen and (max-width: 768px) {
+          display: none;
+        }
+      }
 
-  /* put this above the map drawer but below modal dialogs */
-  z-index: 1250 !important;
+      &__burger {
+        display: none;
+
+        @media only screen and (max-width: 768px) {
+          display: inline-block;
+        }
+      }
+
+      &__picture {
+        width: 40px;
+        height: 40px;
+        border-radius: 50%;
+      }
+
+      &__menu {
+        display: flex;
+        flex-direction: column;
+
+        &__menuitem {
+          display: none;
+          width: 100%;
+
+          @media only screen and (max-width: 768px) {
+            display: flex;
+            flex-direction: column;
+            border-bottom: rgba(0, 0, 0, 0.4) 1px solid;
+          }
+
+          &__links {
+            color: black;
+
+            &:hover {
```

Comment:
Be careful with how heavily you nest these css selectors. This line is like 6-7 levels deep, and it seems generally accepted to keep it 3-4 levels deep (from what I can tell on https://css-tricks.com/forums/topic/sass-best-practices-nesting-more-than-3-levels-deep/).

My reasoning for not nesting too heavily: 
* It adds a bit more CSS that can bloat the CSS over time Ex: this CSS gets precompiled to `.header __navigation __authcontainer __menu __menuItem __links __links:hover`
* Adds a ton of specificity to these selectors that will make it hard to maintain over time

Another resource: https://stackoverflow.com/questions/13805324/how-bad-is-it-in-practice-to-over-nest-selectors-in-sass-scss

Reply:
ahh interesting, I still fairly new to scss. Ill refractor the scss to keep the nesting down.